roles/kolibri/files/kolibri.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kolibri server
+
+[Service]
+User=ideascube
+Group=ideascube
+ExecStart=/home/ideascube/.kolibri/env/bin/python2 /home/ideascube/.kolibri/env/bin/kolibri start
```

Comment:
I would say yes ! Let's start directly with the most advanced version. Ideascube is python 3 anyway